jupyterlab_git/handlers.py

```diff
@@ -262,6 +262,23 @@ def post(self):
         )
 
 
+class GitRemoteAddHandler(GitHandler):
+    """Handler for 'git remote add <name> <url>'."""
+
+    def post(self):
+        """POST request handler to add a remote."""
+        data = self.get_json_body()
+        top_repo_path = data["top_repo_path"]
+        name = data.get("name", "origin")
```

Comment:
The "origin" default is coming up more than once. How about making it a constant and share it?